CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 - [plugin-ext] `ViewContextKeyService#with` method removed. Use `ContextKeyService#with` instead. `PluginViewWidget` and `PluginTreeWidget` inject the `ContextKeyService` rather than `ViewContextKeyService`. [#10546](https://github.com/eclipse-theia/theia/pull/10546)
 
 
+## v1.22.0 - 01/??/2022
+
+[1.22.0 Milestone](https://github.com/eclipse-theia/theia/milestone/30)
+
+- [navigator] added `Open Containing Folder` command [#10523](https://github.com/eclipse-theia/theia/pull/10523)
+
```

Comment:
Minor: The change log section for 1.22.0 has been added above. Please either move your entry or remove it. During our release, an appropriate change log entry for this feature will be created anyway.